src/api/sysinfo.go

```diff
@@ -0,0 +1,70 @@
+package api
+
+import (
+	"github.com/zcalusic/sysinfo"
+	"github.com/shirou/gopsutil/v3/process"
+	"strings"
+)
+
+/* TYPES */
+
+type Distro struct {
+	Name    string
+	Version string
+}
+
+type InitSystem string
+
+func (is InitSystem) String() string {
+	switch is {
+	case SYSTEMD:
+		return "systemd"
+	case RUNIT:
+		return "runit"
+	case OPENRC:
+		return "openrc"
+	}
+	return ""
+}
+
+/* CONSTANTS */
+
+const (
+	SYSTEMD InitSystem = "systemd"
+	RUNIT InitSystem = "runit"
+	OPENRC InitSystem = "openrc"
+	UNKNOWN InitSystem = ""
+)
+
+/* API FUNCTIONS */
+
+func GetCurrentDistro() Distro {
```

Comment:
```suggestion
func GetDistro() Distro {
```

Reply:
I think your suggestion is sensible, although the specification says it should be 'GetCurrentDistro' so we should change the specification too.